bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/system/ITarget.java

```diff
@@ -28,7 +26,7 @@
  */
 public abstract class ITarget implements Cloneable, Serializable {
```

Comment:
I think @fnussber's approach makes sense.  Currently we have a mutable `SPTarget` which contains a mutable `ITarget`.  The end goal is to replace `ITarget` with the core immutable `Target` so this stuff will have to migrate to `Target` at some point.  In the meantime though we need it to exist in `ITarget` anyway.  @tpolecat is target king though, so I'd like to know if he has anything to add.

I know we do a number of model conversions but I think just in AGS calculations?  We never, say, take the science target and convert it from old to new and back to my knowledge.  If we did there would obviously be information loss.